src/components/header/NavigationLanguageMenu.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FC } from 'react';
+import { Box } from '@chakra-ui/react';
```

Comment:
Can you use our component `Box` instead importing it through '@chakra-ui/react'? Thanks